### PR TITLE
RichCommand: HotKey for ToggleShowHiddenItems

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
There was already a hotkey for ToggleShowHiddenItems, but not visible. This pr moves the hotkey into the existing command and adds the information to the button's tooltip.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by yaira2** [comment]
@cinqmilleans can you fix the merge conflict?

**Comment by cinqmilleans** [comment]
It's merged.

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 1]
```diff
@@ -763,28 +763,32 @@
 									Grid.Row="0"
 									Grid.Column="0"
 									VerticalAlignment="Center"
-									Text="{x:Bind Commands.ToggleShowHiddenItems.Label}" />
+									Text="{x:Bind Commands.ToggleShowHiddenItems.Label}"
+									ToolTipService.ToolTip="{x:Bind Commands.ToggleShowHiddenItems.LabelWithHotKey, Mode=OneWay}" />
```
Comment (yaira2) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:767]: TextBlocks don't normally get tooltips, is it really needed here?

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 2]
```diff
@@ -763,28 +763,32 @@
...
 								<ToggleSwitch
 									Grid.Row="0"
 									Grid.Column="1"
 									HorizontalAlignment="Right"
 									AutomationProperties.Name="{x:Bind Commands.ToggleShowHiddenItems.AutomationName}"
 									IsOn="{x:Bind Commands.ToggleShowHiddenItems.IsOn, Mode=TwoWay}"
-									Style="{StaticResource RightAlignedToggleSwitchStyle}" />
+									Style="{StaticResource RightAlignedToggleSwitchStyle}"
+									ToolTipService.ToolTip="{x:Bind Commands.ToggleShowHiddenItems.LabelWithHotKey, Mode=OneWay}" />
```
Comment (yaira2) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:775]: ToggleSwitches don't normally get tooltips, the text is supposed to explain the eaction.

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml (hunk 1 above)
Comment (cinqmilleans) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml]: I removed the tooltip on the label. The tooltip on the button is just the hotkey.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**